instaclustr/resource_bundle_user.go

```diff
@@ -0,0 +1,166 @@
+package instaclustr
```

Comment:
For code re-usability, I suggest moving the CRUD methods to another file and refactoring so that the new data source and resource + the kafka user resource and data sources re-use the CRUD code.